### DISCUSSION TITLE
Where do I add <html data-theme="them-name"> in NEXT 13?

### DISCUSSION THREAD

**Discussion by lucasbarrosomk6** [saadeghi/daisyui#1854] [Q&A] [answered]
Hey everyone, This may seem like a dumb question, but I wanted to make a website using the absolute latest stack. DaisyUI seems like a great fit. I am new to next 13 and the app directory and I am new to DaisyUI. 
however I have a problem. All of the guides on how to add a theme from their provided themes suggest adding the <Html data-theme="lofi"> tag to _document.js or _app.js. But those files do not exist with NEXT 13.  
Can anyone point me in the direction of a guide or just outright tell me where to put this tag?

**Comment by saadeghi** [comment, marked as answer]
In Next.js you can edit the `<html>` tag like this:  
https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required
